sr_robot_launch/launch/srhand.launch

```diff
@@ -32,6 +32,9 @@
   <arg name="robot_state_pub_frequency" default="50"/>
   <!-- The joint state publisher publishing frequency  -->
   <arg name="joint_state_pub_frequency" default="100"/>
+
+  <arg name="variable_controller" default="true"/>
```

Comment:
as in other PR, toni approved this to be default? just want to make sure before merging